### DISCUSSION TITLE
Is there an email list about xTB

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#875] [Q&A] [answered]
Hello,

Is there any email alert/list about the development of xTB? I mean, there are new features added to xTB and will be great if I could be notified about them.

[]'s,
Camps

**Comment by haneug** [comment, marked as answer]
There is an xtb user-list, check out the last point: 
https://www.chemie.uni-bonn.de/grimme/de/software/xtb

**Reply by icamps** [reply]
Thank you very much.
